src/processor/template_processor/base/base_template_constatns.py

```diff
@@ -2,10 +2,13 @@
 from processor.template_processor.azure_template_processor import AzureTemplateProcessor
 from processor.template_processor.google_template_processor import GoogleTemplateProcessor
 from processor.template_processor.terraform_template_processor import TerraformTemplateProcessor
+from processor.template_processor.kubernetes_template_processor import KubernetesTemplateProcessor
+
 
 TEMPLATE_NODE_TYPES = {
     "cloudformation": AWSTemplateProcessor,
     "arm" : AzureTemplateProcessor,
     "deploymentmanager" : GoogleTemplateProcessor,
     "terraform" : TerraformTemplateProcessor,
+    "kubernetesIacFiles" : KubernetesTemplateProcessor
```

Comment:
**kubernetesIacFiles** is not a good name for snapshot type. Change it to **kubernetes**.

Reply:
kubernetes type will use for post-deployment ,I change this type to kubernetesObjectFiles

Reply:
init file exists in kubernetes folder!